avocado/utils/disk.py

```diff
@@ -311,3 +311,115 @@ def get_disk_mountpoint(device):
             if dev == device:
                 return fs_dir
         return None
+
+
+def create_linux_raw_partition(disk_name, size=None, num_of_par=1):
+    """
+    Creates partitions using sfdisk command
+
+    :param disk_name: disk/device name
+    :type disk_name: str
+    :param size: size of partition
+    :type size: str
+    :param num_of_par: Number of partitions to be created
+    :type num_of_par: int
+
+    Returns list of created partitions
+    """
+    if not size:
+        size = get_size_of_disk(disk_name) / 1073741824
+        size = size / num_of_par
+        size = str(size) + "G"
+    partitions = [
+        "size= +" + size if val != 3 else "type=5" for val in range(0, num_of_par + 1)
+    ]
+    disk_partition_file = (
+        "/tmp/creat_partition" + process.run("date '+%d-%m-%y_%T'").stdout_text.strip()
+    )
+    if not os.path.isfile(disk_partition_file):
```

Comment:
where do you keep the partition file ?   better to  add this as parameter to be passed ..  instead hardcoding path

Reply:
got it, you create based on number of partitions given